### PR TITLE
Remove plugins

### PR DESCRIPTION
**Pull request by obecny**
## Which problem is this PR solving?

- removing plugins
- removing meta packages
- updating readme
- updating examples
- downgrading karma from ver. 5 to 4 - it doesn't work locally

### CODE REVIEW

File: examples/dns/package.json [hunk 1]
```diff
@@ -26,13 +26,13 @@
...
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.18.0",
-    "@opentelemetry/exporter-jaeger": "^0.18.0",
-    "@opentelemetry/exporter-zipkin": "^0.18.0",
-    "@opentelemetry/instrumentation": "^0.18.0",
-    "@opentelemetry/instrumentation-dns": "^0.14.0",
-    "@opentelemetry/node": "^0.18.0",
-    "@opentelemetry/tracing": "^0.18.0"
+    "@opentelemetry/api": "^0.18.1",
```
Comment (Flarna) [inline examples/dns/package.json:29]: Should we move api into a peer dependency like in core?

File: examples/mysql/package.json [hunk 2]
```diff
@@ -28,14 +28,14 @@
...
-    "@opentelemetry/instrumentation": "^0.18.0",
-    "@opentelemetry/node": "^0.18.0",
-    "@opentelemetry/plugin-http": "^0.18.0",
-    "@opentelemetry/plugin-mysql": "^0.14.0",
-    "@opentelemetry/tracing": "^0.18.0",
+    "@opentelemetry/api": "^0.18.1",
+    "@opentelemetry/exporter-jaeger": "^0.18.2",
+    "@opentelemetry/exporter-zipkin": "^0.18.2",
+    "@opentelemetry/instrumentation": "^0.18.2",
+    "@opentelemetry/instrumentation-http": "^0.18.2",
```
Comment (Flarna) [inline examples/mysql/package.json:35]: I think moving from plugin-http to instrumentation-http requires to add `registerinitailization()` in sample code.

File: metapackages/auto-instrumentations-node/package.json [hunk 3]
```diff
@@ -43,13 +43,13 @@
     "typescript": "4.1.3"
   },
   "dependencies": {
-    "@opentelemetry/api": "^0.18.0",
-    "@opentelemetry/instrumentation": "^0.18.0",
+    "@opentelemetry/api": "^0.18.1",
```
Comment (Flarna) [inline metapackages/auto-instrumentations-node/package.json:46]: should we move api to peer dependencies like in core?

Comment (vmarchaud) [reply metapackages/auto-instrumentations-node/package.json:46]: Yep but i think it make more sense in another PR

**Comment by vmarchaud** [comment]
I think the history is brokee, there are commit from main on the PR :/

**Comment by obecny** [comment]
> lgtm, just need to fix history

I had cherry picked the changes and created new PR -> https://github.com/open-telemetry/opentelemetry-js-contrib/pull/429

**Pull request closed without merging**